tool/tsh/tsh.go

```diff
@@ -167,6 +169,7 @@ func Run(args []string, underTest bool) {
 	ssh.Arg("[user@]host", "Remote hostname and the login to use").Required().StringVar(&cf.UserHost)
 	ssh.Arg("command", "Command to execute on a remote host").StringsVar(&cf.RemoteCommand)
 	ssh.Flag("port", "SSH port on a remote host").Short('p').Int16Var(&cf.NodePort)
+	ssh.Flag("forward-agent", "Forward agent to target node").Short('A').BoolVar(&cf.ForwardAgent)
```

Comment:
is the long flag consistent with ssh?

Reply:
OpenSSH dose not really have a long flag for `-A`. You can do `-o "ForwardAgent yes"` which we are kind of consistent with.